Choreographer_30FPS/app/src/main/jni/Common/ndk_helper/JNIHelper.cpp

```diff
@@ -0,0 +1,750 @@
+/*
+ * Copyright 2013 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Done.

For consolidating 3 project, I want to keep them separated because those demonstrates different features. Instead, I would make a bug for Android Studio importer to allow 'common' place for duplicated files.